volume/store/store.go

```diff
@@ -104,7 +104,7 @@ func (s *VolumeStore) setNamed(v volume.Volume, ref string) {
 	s.globalLock.Unlock()
 }
 
-func (s *VolumeStore) purge(name string) {
+func (s *VolumeStore) Purge(name string) {
```

Comment:
I think instead of doing this, in the `Remove` function, we should just check if `force` is set, and go ahead and run `purge` if so regardless of error from the driver.

WDYT?